src/lib/radio/radio/radio.html

```diff
@@ -0,0 +1,12 @@
+<template>
+  <div id="radio" class="forge-radio" part="radio">
+    <div id="container" class="container" aria-hidden="true">
+      <div id="background" class="background" part="background"></div>
+      <forge-focus-indicator target=":host" exportparts="indicator:focus-indicator"></forge-focus-indicator>
```

Comment:
Just an FYI for when you merge in the latest changes from the app-bar PR (assuming it is merged before this), the template for `<forge-focus-indicator>` was removed so we can also remove `exportparts` and replace it with just `part="focus-indicator"`